pkg/registry/core/service/rest_test.go

```diff
@@ -518,11 +532,28 @@ func TestServiceRegistryUpdateExternalService(t *testing.T) {
 	if _, _, err := storage.Update(ctx, svc3.Name, rest.DefaultUpdatedObjectInfo(svc3, api.Scheme)); err != nil {
 		t.Fatalf("Unexpected error: %v", err)
 	}
+
+	srv, err := registry.GetService(ctx, svc3.Name, &metav1.GetOptions{})
```

Comment:
Thanks @dcbw for your suggestions! Will address them later.
I should have noticed this when I copied & pasted this block again and again :)